src/object.rs

```diff
@@ -386,6 +386,30 @@ glib_wrapper! {
     }
 }
 
+#[macro_export]
+macro_rules! glib_weak_impl {
```

Comment:
Maybe want to hide this from the docs? This shouldn't really be called directly :)

Reply:
Or do it like for all the other things and make it part of the main macro with a `@` selector for the case. Probably better to keep this consistent.